react/fragments/orderForm.graphql

```diff
@@ -23,6 +23,9 @@ fragment OrderFormFragment on OrderForm {
   }
   marketingData {
     coupon
+    message {
+      code
```

Comment:
Are you not going to send the text (the message itself) here as well?

Reply:
Since the messages used at Figma and the ones that come from Checkout API aren't the same, I think the message's text would be useless.

Reply:
But I think it makes more sense to @augustocb and @davicosta99 define that.